build/configure.js

```diff
@@ -0,0 +1,294 @@
+'use strict';
+// Generate and execute makefile
```

Comment:
Please elaborate it...why we need to generate Makefile while we now have had one manual version, and what's our goal to achieve, by what design with which components?